var/spack/repos/builtin/packages/fastqvalidator/package.py

```diff
@@ -31,26 +31,25 @@ class Fastqvalidator(MakefilePackage):
     homepage = "http://genome.sph.umich.edu/wiki/FastQValidator"
     url      = "https://github.com/statgen/fastQValidator/archive/v0.1.1a.tar.gz"
 
-    version('0.1.1a', '5c5de69527020b72b64f32987409bd12')
-
-    conflicts('%gcc@7:', when='@0.1.1a')  # statgen/fastQValidator#14
+    version('2017-01-10', commit='6d619a34749e9d33c34ef0d3e0e87324ca77f320',
```

Comment:
If ‘10’ is month, you should write it as ‘2017-10-01’, otherwise version comparison won’t work

Reply:
'1' is month, January 10, 2017 is the date of the most recent commit on fastqvalidator github!